build.zig

```diff
@@ -190,13 +190,28 @@ pub fn build(b: *std.build.Builder) void {
         );
 
         const unit_tests = b.addTest("src/unit_tests.zig");
+        unit_tests.addPackage(vsr_package);
         unit_tests.setTarget(target);
         unit_tests.setBuildMode(mode);
         unit_tests.addOptions("vsr_options", options);
         unit_tests.step.dependOn(&tb_client_header_generate.step);
         unit_tests.setFilter(test_filter);
         link_tracer_backend(unit_tests, tracer_backend, target);
 
+        // Turn on test coverage if COV env var is not blank.
+        if (std.process.getEnvVarOwned(b.allocator, "COV")) |coverage| {
+            b.allocator.free(coverage);
+            unit_tests.setExecCmd(&[_]?[]const u8{
+                "kcov",
+                "--clean",
+                "--include-path=src/",
+                "kcov-output",
```

Comment:
Should we add this directory to the .gitignore as well?

Reply:
Yep, it is there